packages/desktop/components/menu-buttons/ProposalDetailsButton.svelte

```diff
@@ -88,11 +88,16 @@
 
 <div class="max-h-7 max-w-9 flex-none flex-initial overflow-visible relative">
     <MeatballMenuButton onClick={modal?.toggle} />
-    <Modal bind:this={modal} position={{ right: '0' }} classes="mt-1.5">
-        <div class="flex flex-col">
-            {#each buttons as button}
-                <MenuItem {...button} />
-            {/each}
-        </div>
-    </Modal>
+    <div class="fixed overflow-visible translate-x-9">
+        <Modal bind:this={modal} position={{ right: '0' }} classes="mt-1.5">
```

Comment:
perhaps we could use the position of the modal to avoid specifying 2 different positions, these 0 and the translate. I am not very opinionated on which one to use, i would just recommend using 1 of them both